### DISCUSSION TITLE
rotating Prop/Entity that can be clamped an an angle and reset back to position

### DISCUSSION THREAD

**Discussion by DecVes** [wiremod/wire#3009] [Suggestions]
This seems like a cool idea to add and It might sound confusing but what I'm asking is for a Prop/Entity that can rotate and be clamped to any angle that can either return back to original position (still clamped) after moved using any 2 keys depending on direction, or stay in the same position and lock at that angle unless commanded by the 2 keys (still clamped). like the wheel but clamped with extra details.
![CLampent](https://github.com/wiremod/wire/assets/115730620/9790f173-4df3-4e8b-b271-87d95db72d2a)

**Comment by DecVes** [comment]
cool beans

**Reply by thegrb93** [reply]
you can do this with gates and gimbal entity
